package/npm/package.json

```diff
@@ -2,13 +2,11 @@
   "name": "elm-format",
   "version": "0.8.5",
   "description": "Install elm-format",
-  "preferGlobal": true,
```

Comment:
`preferGlobal` has been deprecated for a long time, and is even removed in newer npm versions.